packages/design-system-scripts/gulp/build.js

```diff
@@ -90,6 +119,23 @@ async function generateTypeDefinitions(dir, changedPath) {
   );
 }
 
+async function copyCoreTypeDefinitions(dir) {
+  const srcGlob = [
+    path.resolve(`node_modules/${CORE_SOURCE_PACKAGE}/dist/types/index.d.ts`),
+    path.resolve(dir, 'dist', 'types', 'index.d.ts'),
+  ];
+
+  return streamPromise(
+    gulp
+      .src(srcGlob)
+      .pipe(concat('index.d.ts'))
```

Comment:
We will eventually need to think about how this would work with overridden components, but that will be another day